pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -1312,6 +1312,46 @@ func Test_server_SearchByID(t *testing.T) {
 		}
 		return nil
 	}
+
+	/*
+		SearchByID test cases:
+		- Equivalence Class Testing
+			- case 1.1: success search vector from 1000 vectors (type: uint8)
+			- case 1.2: success search vector from 1000 vectors (type: float32)
+			- case 2.1: fail search with non-existent ID form 1000 vectors (type: uint8)
+			- case 2.2: fail search with non-existent ID form 1000 vectors (type: float32)
+		- Boundary Value Testing
```

Comment:
How about removing `from 1000 vectors` from each test case and showing it here instead?
e.g.
```suggestion
		- Boundary Value Testing ( 1000 vectors inserted before a search )
```